src/smk/tool/geomark/tool-geomark.js

```diff
@@ -51,6 +51,8 @@ include.module( 'tool-geomark', [
         function ( smk ) {
             var self = this;
 
+            var CUSTOM_COLOUR = '#ee0077';
```

Comment:
I wanted to make this a const, but this gives a warning as we didn't set ES6 support. (Same for "let".)